hyundai_kia_connect_api/Vehicle.py

```diff
@@ -11,16 +12,27 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+@dataclass
+class Location:
+    lat: decimal = None
+    lon: decimal = None
+
+
 @dataclass
 class TripInfo:
     """Trip Info"""
 
-    hhmmss: str = None  # will not be filled by summary
```

Comment:
Removing fields will break other regions and not work for kia_uvo.   Please try to use the existing fields if possible.